### DISCUSSION TITLE
About ability to put folders into /plugins

### DISCUSSION THREAD

**Discussion by Atreyu-94** [LazyVim/LazyVim#102] [Q&A] [answered]
Hey! By reading docs I supposed it isn't possible, but it could be a good idea for keep plugins in order. Greetings!

**Comment by folke** [comment]
What do you mean?

**Comment by technicalpickles** [comment]
Do you mean like having `lua/plugins/telescope/extensions.lua` or something like that?

**Reply by Atreyu-94** [reply]
Indeed, sorry about my English.

**Comment by luxus** [comment]
u can have folders. but they have to be manually loaded, from plugins/*.lua

**Comment by folke** [comment, marked as answer]
Ok, not I understand. You cam import additional specs in subfolders like:

```
plugins/
  spec1.lua
  spec2.lua
  spec3/
      foo1.lua
      foo2.lua
```

Add `{import = "plugins.spec3"}` to load all the specs under spec3

**Reply by Atreyu-94** [reply]
This is into init.lua or plugins/init.lua ? :)

**Reply by folke** [reply]
In `lua/config/lazy.lua` see https://www.lazyvim.org/configuration/lazy.nvim

Look for `import`

**Reply by Atreyu-94** [reply]
Thanks so much! I'm sorry about delay on answer here but I was studying. Ok, this work very well, I'm going to close this. I really love your plugins!
